website/tests/e2e.fixture.ts

```diff
@@ -91,8 +93,36 @@ export const testSequenceCount: number =
 
 const testUserTokens: Record<string, TokenCookie> = {};
 
+async function ensureKeycloakIsReachable(issuerUrl: string) {
+    try {
+        const response = await axios.get(issuerUrl);
+        if (response.status === 200) {
+            e2eLogger.debug(`Server reachable at ${issuerUrl}.`);
```

Comment:
Maybe this will be too verbose, can you show some logs before/after? I don't know how often this function here is called, maybe I only a few times then it's fine to log this.

Reply:
It is actually called quite often, whenever a user logs in I think. But it's at the 'debug' level so I'd guess it's fine? By default we always run at 'info' anyways. I would leave it, but also happy to drop it if you want to.

Reply:
I have removed it, just in case. 

https://github.com/loculus-project/loculus/pull/2995/commits/2e17caacd9d52d7427521ab1f914624a23c857db